src/styles/blocks/card.scss

```
@@ -0,0 +1,85 @@
+body {
+  font-family: $main-font;
+  font-weight: $main-font-weight;
+  color: $main-font-color;
+  line-height: $main-font-line-height;
+}
+
+.card {
+  background-image: url(../images/imac.jpeg);
+  background-size: 160px 134px;
+  background-repeat: no-repeat;
+  background-position: top 33px left 20px;
+  width: 200px;
+  height: 408px;
+  box-sizing: border-box;
+  padding-top: 207px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__wrap {
+    width: 166px;
```

```suggestion
    width: 100%;
```
With paddings in the parent element it should be the same